pyccel/codegen/printing/ccode.py

```diff
@@ -104,6 +109,82 @@ def _traverse_matrix_indices(self, mat):
     def _print_Module(self, expr):
         return '\n\n'.join(self._print(i) for i in expr.body)
 
+    def _print_While(self,expr):
+        print(expr.test)
+        code = "while (%s)\n{" % self._print(expr.test)
+        code = code + "\n %s" % self._print(expr.body) + "\n}"
+        return (code)
+
+    def _print_If(self, expr):
+        lines = []
+        for i, (c, e) in enumerate(expr.args):
+            #change this line (removes an extra empty else{})
+            if (self._print(e) == ''):
```

Comment:
`self._print(e)` should be stored in a variable at the start of the for loop. That way it can be reused on line 136